src/pixiFunctions/nodeSprites.ts

```diff
@@ -15,6 +15,7 @@ let nodeBoxTextureCache: Map<number, BoxTextures> | undefined
 let arrowTextureCache: Map<number, Texture> | undefined
 let roundedBorderRectCache: Map<RoundedBorderRectCacheKey, BorderRectTextures> | undefined
 
+const transparent = 0x00000000
```

Comment:
What's "0x00000000"? I'm not sure how to think about this variable

Reply:
Ah true, huh.. It just needs to be a unique hex number. Technically that's a hexadecimal color with zero transparency (first 2 numbers after the x), although that hex value results in black as the transparency channel isn't supported.

Reply:
I'll leave a comment in there

Reply:
Got it - thanks for the explanation!